tests/ApplicationTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Casdoor\Tests;
+
+use Casdoor\Auth\Application;
+
+class ApplicationTest
+{
+    public function initConfig()
+    {
```

Comment:
If the test us un-runnable, we usually don't add it. We can use example repo to verify it works